### ISSUE TITLE
Remove ngfactory-

### ISSUE THREAD

**Issue by alexeagle** [angular/angular-bazel-example#449]

**Comment by thesayyn** [comment]
As far as I know, ivy will work with ts_library directly. I saw your work on the main angular repo which contains a ts_plugin for ngtsc. I'm just curious when we'll get rid of @angular/bazel package.

**Comment by alexeagle** [comment]
yeah compiling angular with the `ts_library` rule should make this a lot easier. @alxhub will probably do a refactoring in ngtsc soon which unblocks that work.

**Comment by thesayyn** [comment]
What's the current ETA? May I know that?. Thank you for the information.

**Comment by alexeagle** [comment]
I don't have an ETA - @alxhub has been busy making Ivy work. Maybe we can find someone else to pick it up.
In the meantime this repo is moving to rules_nodejs, and we'll be on Ivy soon which I think will make this problem go away?

Moving to https://github.com/bazelbuild/rules_nodejs/issues/1128 so we don't lose it when I archive this repo
